kubemarine/procedures/check_iaas.py

```diff
@@ -1372,6 +1372,55 @@ def check_ipip_tunnel(group: NodeGroup) -> Set[str]:
                               f"sudo rm -f {ipip_check} {ipip_check}.pid", warn=True)
 
 
+def fs_mount_options(cluster: KubernetesCluster) -> None:
+    with TestCase(cluster, '018', 'System', 'Filesystem mount options') as tc,\
+            suspend_firewalld(cluster):
```

Comment:
No need to suspend firewalld
```suggestion
    with TestCase(cluster, '018', 'System', 'Filesystem mount options'):
```

Reply:
Fixed